release/windows/trezord.nsis

```diff
@@ -165,6 +165,7 @@ SectionEnd
 Section "Install drivers"
   ${If} ${IsWin7}
   DetailPrint "Installing drivers"
+  nsExec::ExecToLog '"$sysdir\cmd.exe" /c "mkdir "%APPDATA%\TREZOR Bridge""'
```

Comment:
You use `%APPDATA%` but `%AppData%` below, is this OK?

Reply:
Oh, that's a leftover from rewriting from NSIS style. But it is ok; in cmd, `%AppData%`, `%APPDATA%`, `%appdata%`, `%aPpDaTa%` etc are the same thing. But I will make the style consistent

Reply:
I have used `APPDATA` everywhere since that is used most often in docs etc